torchdynamo/eval_frame.py

```diff
@@ -104,15 +104,16 @@ def __init__(
         patch_fn()
 
     def __enter__(self):
-        log.warning(
-            (
-                "Using TorchDynamo with a context manager will be deprecated soon."
-                "Please read https://github.com/pytorch/torchdynamo#usage-example "
-                "to use TorchDynamo using an annotation."
-            )
-        )
         if config.raise_on_ctx_manager_usage:
             raise RuntimeError("TorchDynamo is used with a context manager")
```

Comment:
I'd suggest some more context in this error string that perhaps mentions the correct usage.

Reply:
Will do.